### PR TITLE
feature: Display due dates in the ItemViewFragment

### PR DESCRIPTION
**Pull request by YagoGG**

### CODE REVIEW

File: app/src/main/java/com/github/steroidteam/todolist/view/adapter/TodoAdapter.java [hunk 1]
```diff
@@ -37,6 +51,46 @@ public void onBindViewHolder(@NonNull TaskHolder holder, int position) {
...
+            // Make the text's colour light grey as well.
+            dueDate.setSpan(
+                    new ForegroundColorSpan(lightGrey),
+                    0,
+                    dueDate.length(),
+                    Spanned.SPAN_EXCLUSIVE_INCLUSIVE);
+            // Add some margin between the task's body and the due date.
+            holder.taskBody.append("   ");
+            // Insert the span with the due date.
+            holder.taskBody.append(dueDate);
```
Comment (theo-damiani) [inline app/src/main/java/com/github/steroidteam/todolist/view/adapter/TodoAdapter.java:92]: Maybe you can consider having a private method for this.

**Review by theo-damiani — APPROVED** [review]
Great job!
Tested and works well.

Not so much thing to say about your code, and 100% coverage so perfect!

**Review by VPellenc — APPROVED** [review]
Thanks for this feature, the code is well commented.
Once the merge conflicts are solved this PR is good to go!

File: app/src/main/java/com/github/steroidteam/todolist/view/adapter/TodoAdapter.java (hunk 1 above)
Comment (YagoGG) [reply app/src/main/java/com/github/steroidteam/todolist/view/adapter/TodoAdapter.java:92]: Great idea!

**Comment by YagoGG** [comment]
Thank you @theo-damiani and @VPellenc for the reviews!

I just made the suggested changes and rebased the branch. Merging!

**Comment by YagoGG** [comment]
The new layout used by the groupping by date had broken the tests and they required some non-trivial tweaking, but it should all be fine now. :)

**Pull request merged**